documentation/wallet/docs/getting_started/rust.mdx

```diff
@@ -83,11 +82,62 @@ tokio = "1.21.2"
 
 In order to use the library, you need to create an `AccountManager`:
 
-<WarningPasswordStorage/>
+<CodeBlock className="language-rust">
```

Comment:
Not a big fan of this, this would be a nightmare keeping it up to date

Reply:
Overall I think all examples in MD should come from actual code, even if it means adding a new example just for that, but at least it's compiled by the CI and guarantee to always be correct

Reply:
I first wanted to do that, but then I saw the different languages use different sources for their getting started pages (Java uses its README.md, Rust imported the wallet example, Python and Node.js have it in the file directly). So I decided to add the code in the files directly, like 3 of the 4 already did.

I prefer having them in the example directories as well, I can update this PR if you want. Only the READMEs then need to be manually kept up to date.